Scripts/CommonServerPython/CommonServerPython_test.py

```diff
@@ -617,3 +617,25 @@ def test_get_demisto_version(mocker):
         'version': '5.0.0',
         'buildNumber': '50000'
     }
+
+
+def test_parse_date_string():
+    # test unconverted data remains: Z
+    assert parse_date_string('2019-09-17T06:16:39Z') == datetime(2019, 9, 17, 6, 16, 39)
+
+    # test unconverted data remains: .22Z
+    assert parse_date_string('2019-09-17T06:16:39.22Z') == datetime(2019, 9, 17, 6, 16, 39, 220000)
+
+    # test time data without ms does not match format with ms
+    assert parse_date_string('2019-09-17T06:16:39Z', '%Y-%m-%dT%H:%M:%S.%f') == datetime(2019, 9, 17, 6, 16, 39)
+
+    # test time data with timezone Z does not match format with timezone +05:00
+    assert parse_date_string('2019-09-17T06:16:39Z', '%Y-%m-%dT%H:%M:%S+05:00') == datetime(2019, 9, 17, 6, 16, 39)
```

Comment:
wouldn't we want to convert it to UTC maybe?

Reply:
not sure i understand what you mean

Reply:
this time is represented in local time rather than in UTC, if we want to calculate the UTC we will decrease 5 hours of the time we got